learning/tour-of-beam/frontend/lib/components/footer.dart

```diff
@@ -31,12 +30,25 @@ class Footer extends StatelessWidget {
   Widget build(BuildContext context) {
     return _Body(
       child: Wrap(
-        spacing: TobSizes.size16,
+        alignment: WrapAlignment.spaceBetween,
         crossAxisAlignment: WrapCrossAlignment.center,
         children: [
-          const _ReportIssueButton(),
-          const _PrivacyPolicyButton(),
-          const Text('ui.copyright').tr(),
+          Wrap(
+            spacing: BeamSizes.size16,
+            crossAxisAlignment: WrapCrossAlignment.center,
+            children: [
+              const _ReportIssueButton(),
+              const _PrivacyPolicyButton(),
+              const Text('ui.copyright').tr(),
+            ],
+          ),
+          // TODO(nausharipov): get version
+          Text(
+            '${'ui.builtWith'.tr()} 2.40.0',
```

Comment:
Avoid text that looks proper but does not work. A good example is `TODO: Version` or `7.77.0`, so we are sure to not release a fake to production.

Reply:
Done.